keps/sig-ui/5008-kep-template/kep.yaml

```diff
@@ -0,0 +1,37 @@
+title: Kubernetes Desktop
+kep-number: 5008
+authors:
+  - "@joaquimrocha"
+owning-sig: sig-ui
+participating-sigs:
+status: provisional
```

Comment:
please fill all of this kep.yaml

Reply:
You mean the participating-sigs, right? I think only the sig-ui makes sense here. I will add it as the participating sig in a new PR.